src/main/java/uk/nhs/tis/trainee/notifications/model/NotificationType.java

```diff
@@ -44,6 +44,7 @@ public enum NotificationType {
   LTFT("less-than-full-time"),
   NON_EMPLOYMENT("non-employment"),
   PLACEMENT_INFORMATION("placement-information"),
+  PLACEMENT_USEFUL_INFORMATION("placement-useful-information"),
```

Comment:
I think it would be this key name  `PLACEMENT_USEFUL_INFORMATION` need to be shorten if you want to change.